client/blocks/product-purchase-features-list/google-vouchers.jsx

```diff
@@ -5,18 +5,30 @@
  */
 
 import React from 'react';
+import { useTranslate } from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
 import GoogleVoucherDetails from 'my-sites/checkout/checkout-thank-you/google-voucher';
+import PurchaseDetail from 'components/purchase-detail';
 import QuerySiteVouchers from 'components/data/query-site-vouchers';
 
 export default ( { selectedSite } ) => {
+	const translate = useTranslate();
 	return (
 		<div className="product-purchase-features-list__item">
 			<QuerySiteVouchers siteId={ selectedSite.ID } />
-			<GoogleVoucherDetails selectedSite={ selectedSite } />
+			<PurchaseDetail
+				alt=""
+				id="google-credits"
+				icon={ <img alt="" src="/calypso/images/illustrations/google-adwords.svg" /> }
```

Comment:
Shouldn't we provide the `alt` attribute here and above?

Reply:
removed as it is an unused attribute.

Reply:
Apologies if I missed something but I still see an empty `alt=""` here.